### DISCUSSION TITLE
do we need an Opcache/JIT while using FrankenPHP with Laravel?

### DISCUSSION THREAD

**Discussion by arlanram** [php/frankenphp#877] [Q&A]
as FrankenPHP used to run in memory do we need to compile Laravel code using opcache and JIT?

we think that we don't have to use opcache, jit because Laravel code already stays compiled in memory.

please explain these things! thanks a lot🙏

**Comment by dunglas** [comment]
Using opcache is indeed less useful in worker mode, but allows workers to restart faster after when they reboot.

JIT may still help, but I haven't performed any benchmarks.
